Documentation program

Calm-Rock opened the pull request:
This is the PR for the Technical Documentation Program. 


Relevant Issue: [here](https://github.com/orgs/SigNoz/projects/12?pane=issue&itemId=79035926)

prashant-shahi commented on docs/userguide/logs_troubleshooting.md:
```diff
@@ -0,0 +1,45 @@
+
+
+## K8s attribute filtering issue in logs.
+In the recent release some users who are using SigNoz charts(0.9.1 - 0.10.1) are facing issues querying the following selected fields.
+
+- k8s_container_name
+- k8s_namespace_name
+- k8s_pod_name
+- k8s_container_restart_count
+- k8s_pod_uid
```
```suggestion
In the SigNoz charts releases `v0.9.1`, `v0.10.0` and `0.10.1`, some users who are facing
issues querying the following selected fields.

- `k8s_container_name`
- `k8s_namespace_name`
- `k8s_pod_name`
- `k8s_container_restart_count`
- `k8s_pod_uid`
```

prashant-shahi commented on docs/userguide/logs_troubleshooting.md:
```diff
@@ -0,0 +1,45 @@
+- k8s_pod_uid
+
+
+If you have converted any of the above fields to `selected` fields, and you notice that when you filter using the above fields it results in empty data then you will have to perform the following steps.
```
```suggestion
If you have included any of the above to `selected` fields, and you get empty data
when you filter using those fields then you will have to perform the following steps.
```

prashant-shahi commented on docs/userguide/logs_troubleshooting.md:
```diff
@@ -0,0 +1,45 @@
+
+If you have converted any of the above fields to `selected` fields, and you notice that when you filter using the above fields it results in empty data then you will have to perform the following steps.
+
+- Exec into your clickhouse container
```
kubectl command to exec into clickhouse container

```bash
kubectl exec -n platform -it chi-my-release-clickhouse-cluster-0-0-0 -- sh
```

prashant-shahi commented on docs/userguide/logs_troubleshooting.md:
````diff
@@ -0,0 +1,45 @@
+- Exec into your clickhouse container
+- Run the following queries
+    
+    ```yaml
+    use signoz_logs;
+    
+    show create table logs;
+    ```
````
verify that indentation in build is correct

prashant-shahi commented on docs/userguide/logs_troubleshooting.md:
````diff
@@ -0,0 +1,45 @@
+    show create table logs;
+    ```
+    
+- For the corresponding field, you will find a materialized column and an index. Ex:- For `k8s_namespace_name` you will have `k8s_namespace_name String MATERIALIZED attributes_string_value[indexOf(attributes_string_key, 'k8s_namespace_name')] CODEC(LZ4)`     and index `INDEX k8s_namespace_name_idx k8s_namespace_name TYPE bloom_filter(0.01) GRANULARITY 64`
````
```suggestion
- For the corresponding field, you will find a materialized column and an index.
For example: `k8s_namespace_name` you will have `k8s_namespace_name String
MATERIALIZED attributes_string_value[indexOf(attributes_string_key,
'k8s_namespace_name')] CODEC(LZ4)` and index `INDEX k8s_namespace_name_idx
k8s_namespace_name TYPE bloom_filter(0.01) GRANULARITY 64`
```

prashant-shahi reviewed (changes requested):
left few comments

prashant-shahi reviewed (approved):
:shipit: LGTM

Calm-Rock committed:
feat: documentation program

Calm-Rock committed:
feat: documentation program

Calm-Rock committed:
added content for the expert contributions program

Calm-Rock committed:
update: updated content

Calm-Rock committed:
Update page.tsx

makeavish commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,111 @@
+
+### Advanced Dashboards:
+
+Create sophisticated dashboards tailored for specific technologies, industries, or use cases.
```
Can we hyperlink dashboards?
https://github.com/SigNoz/dashboards

makeavish commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,111 @@
+[![Apply for SigNoz Technical Writing Program](/img/ecp-cta.webp)](https://forms.gle/f2TogL5GZ7CZ7rUY9)
+
+
+## Resources
```
Are we going to add something here?

makeavish commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,111 @@
+
+Click the "Apply Now" button below 👇 and complete the form. If your profile aligns with our requirements, our team will reach out to you with the next steps.
+
+[![Apply for SigNoz Technical Writing Program](/img/ecp-cta.webp)](https://forms.gle/f2TogL5GZ7CZ7rUY9)
```
Apply for SigNoz Experts Contribution Program

Calm-Rock committed:
Update page.tsx

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+**Monetary Compensation:** Receive competitive payments for your high-quality contributions to SigNoz.
+
+**Exclusive Badges:** Earn SigNoz Expert Badges as proof of your work and expertise. Display them on your professional profiles, resumes, or portfolios.
```
```suggestion
**Exclusive Badges:** Earn SigNoz Expert Badges as proof of your work and expertise. Display them on your professional profiles, resumes, or portfolios.
```

How are we planning to do this? How will someone display the badge on sth like linkedin? Let's not mention this if there is no clear plan.

Calm-Rock replied:
> How are we planning to do this? How will someone display the badge on sth like linkedin? Let's not mention this if there is no clear plan.

Was planning to do this in a similar way that PostHog does. This will require us to create a page for it and have links for individual contributors which they can use on LinkedIn or anywhere else they want.

Removing this for now. Will pick it up in the second interaction.

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+&nbsp;
+
+### Advanced Dashboards:
```
```suggestion
### Dashboards:
```

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+### Advanced Dashboards:
+
+Create sophisticated dashboards tailored for specific technologies, industries, or use cases.
```
```suggestion
Create sophisticated dashboards tailored for specific technologies, industries, or use cases.
```

sophisticated is not going well here imo. Maybe "Create dashboards for monitoring specific technologies, industries or use cases."

Also can we share some examples of dashboards here:

Some examples of dashboards are:
- Hostmetrics dashboard (link to docs)
- Redis dashboard (link to docs)

You can check out the existing dashboards here.

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+Develop in-depth technical documentation, tutorials, and product explanations and also test new documentation created by other contributors.
+
+Compensation Range: **Upto $500/contribution**
```
```suggestion
Compensation Range: **Upto $500/contribution**
```

Shouldn't this be as much as dashboards. As in upto $300/contribution.

Calm-Rock replied:
Cool. Updated it.

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+### Recognition and Payment:
+
+○ Earn your SigNoz Expert Badge upon successful contribution.
```
```suggestion
○ Earn your SigNoz Expert Badge upon successful contribution.
```

Same feedback as earlier comment. How are we planning to do this?

Calm-Rock replied:
Removing this for now. This will be part of the second iteration.
Created an issue for second iteration here - https://github.com/SigNoz/signoz-web/issues/837

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+○ Earn your SigNoz Expert Badge upon successful contribution.
+
+○ Receive prompt payment for your contributions.
```
```suggestion
○ Receive prompt payment for your contributions.
```

Payments will be done based on our schedule currently. Check payment faqs created by Yuvraj.

Calm-Rock replied:
Have added clear details about the payment process.

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+# SigNoz Expert Contributors Program
+
+The SigNoz Expert Contributors Program is an exclusive initiative for seasoned professionals in the observability domain. 
```
@Calm-Rock can you check the intro and why write technical content for us from here: https://signoz.io/technical-writer-program/

The current write-up sounds bland and doesn't feel inspiring. Also include a few lines about SigNoz as we have done in the write up for Technical Writer Program. Assume that readers landing here might not have heard about SigNoz.

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+○ Our team will review your expertise and align your skills with the task needs.
+
+### Task Assignment
```
```suggestion
### Task Assignment
```

Task and assignment mean the same thing, right?

Calm-Rock replied:
I meant Assigning the task. Have updated it to "Task Assigned"

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+
+### Contribution and Review
+
+○ Conduct independent research and testing, then submit a pull request (PR) to the appropriate repository.
```
```suggestion
○ Conduct independent research and testing, then submit a pull request (PR) to the appropriate repository.
```

Let's make these things more detailed in the second iteration

Calm-Rock replied:
Cool. Have created an issue for things to take up in second iteration - https://github.com/SigNoz/signoz-web/issues/837

ankit01-oss commented on app/signoz-expert-contributors-program/page.tsx:
```diff
@@ -0,0 +1,106 @@
+# SigNoz Expert Contributors Program
+
+The SigNoz Expert Contributors Program is an exclusive initiative for seasoned professionals in the observability domain. 
+If you're an expert eager to showcase your skills, contribute to a leading open-source project, and get compensated for your expertise, this program is for you.
```
```suggestion
Looking for an opportunity to showcase your expertise while contributing to a top open-source project? Then this program might just be what you’ve been waiting for.
```

Calm-Rock committed:
incorporated feedback

UnCool-0x committed:
Documentation program (#838)

* feat: add delayed close functionality to product and resources dropdowns (#792)

* launch week thurs (#785)

* update

* update

* merged

* add blog

* add blog

---------

Co-authored-by: Ankit Anand <83692067+ankit01-oss@users.noreply.github.com>

* ankit chat blog (#793)

* update

* update

* merged

* add

* Changes for day 4 (#794)

Co-authored-by: ankit01-oss <>

* feat: docs for logs-v2 (#790)

* feat: docs for logs-v2

* feat: minor fix

* launch week fri (#786)

* update

* update

* merged

* add blog

* Changes for day 5 (#796)

Co-authored-by: ankit01-oss <>

* fix: fix youtube playback error (#798)

* fix issues in otel nodejs (#799)

* update

* update

* merged

* minor fixes

* Removed launch week banner and add launch week link in footer (#801)

Co-authored-by: ankit01-oss <>

* Fix typos in collect_logs_from_file.mdx (#802)

* fix: typo Azure Blob Storage SideNav (#800)

* fix: layout fix for cards (#803)

* fix feed xml (#791)

* update

* update

* merged

* fix feed.xml invalid link issue

* Added case study for Cedana (#809)

* feat: migration guide for clickhouse queries (#812)

* upload articles 25 09 (#811)

* update

* update

* merged

* update

* update

* Add Articles

* add more

* feat: migration guide 0.55 (#813)

* feat: migration guide 0.55

* fix: minor fixes

* fix: minor fixes

* fix: minor fixes

* fix: minor fixes

* chore: minor v0.55 migration changes (#814)

* Fix typos on PostgreSQL integrations and otel collector docs (#773)

* Scroll Form FAQs (#797)

* update

* update

* merged

* trying

* Add what you looking for form on home

* added form to faq pages

* fix issue

* final design

* Update options

* Case study with Kiwi (#815)

* update: updated vector logs docs (#807)

* update: updated vector logs docs

* fix: removing yarn file

* fix: removed yarn lock file

* fix: fixed the feedback component location (#818)

* Minor edit (#819)

Co-authored-by: ankit01-oss <>

* feat: add Trieve search and AI Chat (RAG) (#816)

Co-authored-by: Sara Vieira <hey@iamsaravieira.com>

* fix: add links in migration guide (#822)

* fix: removed info block from self host instructions (#824)

* Added case study for bands (#825)

* Added case study for bands

* removed image

* added image

* Update data/case-study/bands.mdx

---------

Co-authored-by: ankit01-oss <>

* feat: mobile-instrumentation (#808)

* feat: mobile-instrumentation

* minor changes

* notice

* new tag

* feat: removed self hosted signoz

* fix github urls

* feat: removed new doc tag

* feat: region

* feat: removed smaller table

* fix: removed new tag from mobile monitoring (#827)

* fix structure data (#828)

* update

* update

* merged

* update

* update

* fix

* fix feed.xml

* feat: add search button to mobile (#830)

* feat: add search button to mobile

* fix: add gap between search and hamburger

* fix: search button hidden when drop down is active

* fix: moved mobile monitoring section (#832)

* added BAA in teams plan (#834)

* feat(perf): optimized large assets (#835)

- removed un used fonts for new theme
- removed un used CSS
- optimized image asset worth ~3mb

* chore: lock file

---------

Co-authored-by: harshbutfairx <itsharshnarayan@gmail.com>
Co-authored-by: Yuvraj Singh Jadon <yuvraajsj18@gmail.com>
Co-authored-by: Ankit Anand <83692067+ankit01-oss@users.noreply.github.com>
Co-authored-by: Nityananda Gohain <nityanandagohain@gmail.com>
Co-authored-by: Jarkko Saltiola <jarkko.saltiola@gmail.com>
Co-authored-by: konnta0 <68390856+konnta0@users.noreply.github.com>
Co-authored-by: Prashant Shahi <prashant@signoz.io>
Co-authored-by: Austin Nazworth <98629900+anazworth@users.noreply.github.com>
Co-authored-by: CheetoDa <31571545+Calm-Rock@users.noreply.github.com>
Co-authored-by: skeptrune <nicholas.khami@gmail.com>
Co-authored-by: Sara Vieira <hey@iamsaravieira.com>
Co-authored-by: Pranay Prateek <pranay@signoz.io>
Co-authored-by: Sudeep MP <sudipmelepunathilmp@gmail.com>

Calm-Rock commented:
Resolved the merge conflicts in this PR - #839
